docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -0,0 +1,21 @@
+== Installing Jenkins plugins
+Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be specified in Ansible playbook(???) or using the Jenkins user interface.
```

Comment:
@laurafitzgerald Good point, it won't just extend it, that's sad. I suppose we could suggest copying the value over into the vars file and then modifying? That way they always have the defaults unmodified.